.gitlab/check_merge/do_not_merge.yml

```
@@ -1,18 +1,26 @@
 do-not-merge:
   stage: check_merge
-  image: registry.ddbuild.io/ci/datadog-agent-buildimages/linux-glibc-2-17-x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  image: registry.ddbuild.io/ci/datadog-agent-buildimages/linux-glibc-2-17-x64$CI_IMAGE_LINUX_GLIBC_2_17_X64_SUFFIX:$CI_IMAGE_LINUX_GLIBC_2_17_X64
   tags: ["arch:amd64"]
   rules:
   - !reference [.on_dev_branches]
   - when: always
   needs: []
   script:
       - |
-          if [ ! -z "$DATADOG_AGENT_BUILDIMAGES_SUFFIX" ] ||
-             [ ! -z "$DATADOG_AGENT_WINBUILDIMAGES_SUFFIX" ] ||
-             [ ! -z "$DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX" ] ||
-             [ ! -z "$DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX" ] ||
-             [ ! -z "$DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX" ] ||
+          if [ ! -z "$CI_IMAGE_BTF_GEN_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_DEB_X64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_DEB_ARM64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_DEB_ARMHF_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_DOCKER_X64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_DOCKER_ARM64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_GITLAB_AGENT_DEPLOY_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_GLIBC_2_17_X64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_GLIBC_2_23_ARM64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_RPM_X64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_RPM_ARM64_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_RPM_ARMHF_SUFFIX" ] ||
+             [ ! -z "$CI_IMAGE_LINUX_WIN_LTSC2022_X64_SUFFIX" ] ||
              [ ! -z "$TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX" ]; then
```

Pro tip: the opposite of `-z` (`the length of STRING is zero`) is `-n` (`the length of STRING is nonzero`).

```suggestion
          if [ -n "$CI_IMAGE_BTF_GEN_SUFFIX" ] ||
             [ -n "$CI_IMAGE_DEB_X64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_DEB_ARM64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_DEB_ARMHF_SUFFIX" ] ||
             [ -n "$CI_IMAGE_DOCKER_X64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_DOCKER_ARM64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_GITLAB_AGENT_DEPLOY_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_GLIBC_2_17_X64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_GLIBC_2_23_ARM64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_RPM_X64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_RPM_ARM64_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_RPM_ARMHF_SUFFIX" ] ||
             [ -n "$CI_IMAGE_LINUX_WIN_LTSC2022_X64_SUFFIX" ] ||
             [ -n "$TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX" ]; then
```